### DISCUSSION TITLE
Chia Wallet Can Not Receive Transferred Balance

### DISCUSSION THREAD

**Discussion by Ratchanida** [Chia-Network/chia-blockchain#6965] [Support]
Yesterday, I initiated a transaction transfer to my other chia wallet internally within Chia Network.  The history status indicated "confirmed" but I still see 0 balance. It's has been over 24 hours.  (I created a second wallet, supposedly to be my cold wallet) 
I make sure the wallets are all synced.  I also tried to close my Chia Blockchain and reopen, still it showed no incoming transaction.  Please help.

**Comment by loppefaaret** [comment]
is the wallet synced to the same height as the node ? and is the node synced ?

**Reply by Ratchanida** [reply]
The issue has resolved. I exited and reopened Chia Blockchain multiple times and saw the incoming transaction. :) Thanks

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
